spec/models/batch_spec.rb

```diff
@@ -17,4 +17,23 @@
 
   end
 
+  it 'should have the property att_timestamp_unixtime_ms {base name: "att_timestamp_unixtime_ms" }' do
+    device_info = MParticle::DeviceInformation.new
+    now = Time.now.to_i
```

Comment:
this is actually an existing bug in the code - this should be something like  `(Time.now.to_f * 1000).to_i` to get milliseconds rather than seconds - can you update it here and elsewhere?